ui/tests/pages/components/control-group-success.js

```diff
@@ -1,7 +1,7 @@
 import { isPresent, fillable, clickable } from 'ember-cli-page-object';
 
 export default {
-  showsJsonViewer: isPresent('[data-test-json-viewer]'),
+  showsJsonViewer: isPresent('[data-test-component="json-editor"]'),
```

Comment:
because of changing the component to glimmer, this was wrapped around the component, but now that it's glimmer that doesn't work.